common/scylla_cloud_io_setup

```diff
@@ -423,6 +423,21 @@ class aws_io_setup(cloud_io_setup):
             self.disk_properties["read_bandwidth"] = 3088599296 * nr_disks
             self.disk_properties["write_iops"] = 239549 * nr_disks
             self.disk_properties["write_bandwidth"] = 2302438912 * nr_disks
+        elif self.idata.instancetype == "i7ie.large":
+            self.disk_properties["read_iops"] = 58449
+            self.disk_properties["read_bandwidth"] = 574854656
+            self.disk_properties["write_iops"] = 47145
+            self.disk_properties["write_bandwidth"] = 253132917
+        elif self.idata.instancetype in ("i7ie.xlarge", "i7ie.2xlarge"):
+            self.disk_properties["read_iops"] = 117257 * nr_disks
+            self.disk_properties["read_bandwidth"] = 1148572714 * nr_disks
+            self.disk_properties["write_iops"] = 94180 * nr_disks
+            self.disk_properties["write_bandwidth"] = 505684885 * nr_disks
+        elif self.idata.instancetype in ("i7ie.3xlarge", "i7ie.6xlarge", "i7ie.12xlarge", "i7ie.18xlarge", "i7ie.24xlarge", "i7ie.48xlarge"):
+            self.disk_properties["read_iops"] = 352834 * nr_disks
+            self.disk_properties["read_bandwidth"] = 3422623232 * nr_disks
+            self.disk_properties["write_iops"] = 119327 * nr_disks
+            self.disk_properties["write_bandwidth"] = 1526442410 * nr_disks
```

Comment:
Where do these numbers come from?

Please add to patch changelog.

Reply:
Unlike other clouds like Azure, EC2 does not have IOPS and bandwidth information on their document.
So all preset values are measured by iotune on target instances instead of referenced from the document.
I measured 3 times on each instance types and used average of the result.

Added same description on the changelog.

Reply:
ok

Reply:
previously @xemul had some cavets on how we calculate the numbers for i4i.
I want to make sure we're doing it transparently this time.
Let's have it approved from @xemul as well.